app/scripts/migrations/048.js

```diff
@@ -87,5 +88,71 @@ function transformState (state = {}) {
     })
   }
 
+  // 6.  Convert address book keys from decimal to hex
+  const addressBook = state.AddressBookController?.addressBook || {}
+  Object.keys(addressBook).forEach((networkKey) => {
+    if ((/^\d+$/ui).test(networkKey)) {
```

Comment:
Wouldn't this exclude the very keys we want to test? e.g. `0x1`, because `x` isn't a digit

Reply:
For posterity: we're trying to find keys that we want to convert, namely digit-only keys.